src/execution_utils.jl

```diff
@@ -196,6 +196,7 @@ end
             launch(queue, signal, f, groupsize, gridsize, ($(arg_ptrs...),))
         end
     end).args)
+    Core.println(ex)
```

Comment:
Part of my debugging efforts, since the segfault seems to occur somewhere within this generated function's emitted expression.